src/module/apps/sidebar/chat-log.ts

```diff
@@ -8,9 +8,61 @@ import { TokenDocumentPF2e } from "@scene";
 import { CheckPF2e } from "@system/check/index.ts";
 import { looksLikeDamageRoll } from "@system/damage/helpers.ts";
 import { DamageRoll } from "@system/damage/roll.ts";
-import { fontAwesomeIcon, htmlClosest, htmlQuery, objectHasKey } from "@util";
+import { fontAwesomeIcon, htmlClosest, htmlQuery, objectHasKey, sluggify } from "@util";
+import { ActionUseOptions } from "@actor/actions/index.ts";
 
 class ChatLogPF2e extends ChatLog<ChatMessagePF2e> {
+    static readonly #actPattern = new RegExp(/\s*\/(act) (?<slug>[-a-z]+)\s*(?<options>.+)?/, "i");
+
+    static override parse(message: string): [string, RegExpMatchArray] {
+        const match = message.match(this.#actPattern);
+        if (match) {
+            return ["macro", match]; // abuse the "macro" chat command since that suppresses message creation
+        }
+        return super.parse(message);
+    }
+
+    /**
+     * Process messages which execute a macro.
+     * @param {string} command  The chat command typed.
+     * @param {RegExpMatchArray} match  The RegExp matches.
+     * @private
+     */
+    override _processMacroCommand(command: string, match: RegExpMatchArray): unknown {
+        if (match && match.length > 2 && match[1] === "act" && match.groups?.slug) {
```

Comment:
```suggestion
        if (match[1] === "act" && match.groups?.slug) {
```

Reply:
Moved to https://github.com/foundryvtt/pf2e/pull/12772 instead.